docs/source/tutor/_hello_world.ipynb

```diff
@@ -0,0 +1,717 @@
+   "metadata": {},
+   "source": [
+    "We can now load the `quick_hello.py` file located in the user directory and execute its contents in the current Python environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading 'Hello, World!' example.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%run -i ~/bluesky/user/quick_hello.py"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: The `Loading 'Hello, World!' example.` text came from a `print()` statement in the file.\n",
+    "\n",
+    "This also loads `hello_world()`, a demonstration bluesky *plan*.  We can run the `hello_world()` function in the `RE` RunEngine instance, which will execute the corresponding experimental plan. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\n",
+      "Transient Scan ID: 1     Time: 2023-04-21 07:46:14\n",
+      "Persistent Unique Scan ID: 'ac400921-948c-453a-814d-f8720ba7b783'\n",
+      "New stream: 'primary'\n",
+      "+-----------+------------+------------+\n",
+      "|   seq_num |       time |      hello |\n",
+      "+-----------+------------+------------+\n",
+      "|         1 | 07:46:14.4 |          1 |\n",
+      "+-----------+------------+------------+\n",
+      "generator count ['ac400921'] (scan num: 1)\n",
+      "\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "('ac400921-948c-453a-814d-f8720ba7b783',)"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(hello_world())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can retrieve data for **the most recent run** stored in the `cat` databroker instance, and assign it to the a variable `run`. Since we have created a temporary catalog, the data is only available until the end of the IPython session.\n",
+    "\n",
+    "<details>\n",
+    "<summary>More about retrieving data</summary>\n",
+    "\n",
+    "When working with a regular (\"not temporary\") catalog, more details about working with the data can be found here: \n",
+    "- <a ref=https://bcda-aps.github.io/bluesky_training/howto/_after_measurement.html> How to access the data later, after the measurement.</a>\n",
+    "- <a ref=https://bcda-aps.github.io/apstools/latest/examples/ho_search_databroker.html> How to find Bluesky data.</a>\n",
+    "</details>\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = cat[-1]\n",
+    "header=run.metadata\n",
```

Comment:
To avoid confusion with the databroker v1 `header` term, let's use `metadata = run.metadata`

This will be consistent with the messaging that Bluesky makes  _metadata_ integral to the process.  Search relies on finding and matching _metadata_.